packages/sdk-effects/oidc/src/index.ts

```diff
@@ -1,2 +1,8 @@
-export * from './lib//authorize.effects.js';
+/*
+ * Copyright (c) 2025 Ping Identity Corporation. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+export * from './lib/authorize.effects.js';
```

Comment:
per our conversation on Slack, this technically, is an api addition.

So, the question is why?

Reply:
Ok so I see what happened here, we kind of reorganized this. it's still an api addition though.

Reply:
Added minor changeset

Reply:
Specifically referring to the line in this comment. why are we adding authorize.effects.js to the index.ts file?

This is the api change, and i'm not seeing us use this outside of here right now.

is this because we're going to consume functions from here in the future?